exovetter/const.py

```diff
@@ -29,4 +29,10 @@
 """PPM"""
 
 frac_amp = u.dimensionless_unscaled
-"""Frac amp"""
+"""PPM"""
```

Comment:
`frac_amp` is PPM too or is this a typo?

Reply:
Good catch, that should be Fractional Amplitude.